main.go

```diff
@@ -39,17 +40,30 @@ const (
 )
 
 func main() {
-	var filename string
-	var printVersion bool
-	var debug bool
-	var once bool
+
+	var (
+		filename     string
+		printVersion bool
+		debug        bool
+		once         bool
+		logfmt       string
+	)
 
 	flag.StringVar(&filename, "config", "", "path to config file")
 	flag.BoolVar(&printVersion, "version", false, "show version")
 	flag.BoolVar(&debug, "debug", false, "enable debugging mode")
 	flag.BoolVar(&once, "once", false, "run git2consul once and exit")
+	// allow switching logformat. Structured output helps with parsers
+	flag.StringVar(&logfmt, "logfmt", "text", "specify log format [text | json] ")
 	flag.Parse()
 
+	// Init checks
+	if len(filename) == 0 {
+		log.Errorf("No configuration file provided")
```

Comment:
I think this should use `log.Error` instead of `log.Errorf`.

Reply:
thank you @seanmalloy fixed this